docs/index.rst

```diff
@@ -9,8 +9,9 @@ R5py
    :caption: Contents
 
    installation
+   configuration
    basic-usage.ipynb
-   CONTRIBUTING.md
+   CONTRIBUTING
```

Comment:
Are you sure this works without the `.md` file extension? Could well be. Mostly asking to update my knowledge if this is the case.